apps/hoa-recipes/src/app/app.component.html

```diff
@@ -1,2 +1,3 @@
 <hoa-recipes-feature-title-bar></hoa-recipes-feature-title-bar>
+<hoa-recipes-feature-recipe-list></hoa-recipes-feature-recipe-list>
```

Comment:
you can use self-closed tags
```suggestion
<hoa-recipes-feature-recipe-list />
```